packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -537,10 +539,46 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
     /// @notice Returns the required bond for a given move kind.
     /// @param _position The position of the bonded interaction.
     /// @return requiredBond_ The required ETH bond for the given move, in wei.
-    function getRequiredBond(Position _position) public pure returns (uint256 requiredBond_) {
-        // TODO(client-pod#551): For now use a non-zero bond amount to unblock functional tests.
-        _position;
-        requiredBond_ = 0.01 ether;
+    function getRequiredBond(Position _position) public view returns (uint256 requiredBond_) {
+        int256 depth = int256(uint256(_position.depth()));
+        if (depth > int256(MAX_GAME_DEPTH)) revert GameDepthExceeded();
+
+        // Values taken from Big Bonds v1.5 (TM) spec.
+        int256 assumedBaseFee = 200 gwei;
+        int256 baseGasCharged = 400_000;
+        int256 highGasCharged = 200_000_000;
+
+        // Goal here is to compute the fixed multiplier that will be applied to the base gas
+        // charged to get the required gas amount for the given depth. We apply this multiplier
+        // some `n` times where `n` is the depth of the position. We are looking for some number
+        // that, when multiplied by itself `MAX_GAME_DEPTH` times and then multiplied by the base
+        // gas charged, will give us the maximum gas that we want to charge.
+        // We want to solve for (highGasCharged/baseGasCharged)^(1/MAX_GAME_DEPTH).
+        // We know that a^(b/c) is equal to e^(ln(a) * (b/c)).
+        // We can compute e^(ln(a) * (b/c)) quite easily with FixedPointMathLib.
```

Comment:
Assuming you mean exponentiation (`**`) rather than xor (`^`)?

Reply:
Correct